src/RestApiContext.php

```diff
@@ -225,10 +225,28 @@ private function sendRequest($method, $url, $body = null)
         }
     }
 
+    /**
+     * @param string                $method
+     * @param string                $url    With or without host
+     * @param string|resource|array $body
+     */
     private function createRequest($method, $url, $body = null)
     {
+        if (!$this->hasHost($url)) {
+            $url = $this->httpClient->getBaseUrl() . $url;
```

Comment:
I guess we will have to deal with ending slash in baseUrl and/or starting slash in $url argument to avoid 
```
http://verylastroom.com//hello.json
http://verylastroom.comhello.json
```